config/routes.rb

```diff
@@ -19,6 +19,8 @@
       resources :children, controller: :units_children, only: :index
     end
     resources :tickets, only: %i[show new create] do
+      post '/resolution', to: 'tickets#resolution', as: :resolution
+      get  '/followed_new', to: 'tickets#followed_new', as: :followed
```

Comment:
```suggestion
      post  '/follow_up', to: 'tickets#followed_up'
```
У екшені створювати тікет і одразу редіректити на його едіт.